### PR TITLE
Entity mappings

### PR DESCRIPTION
**Pull request by Runemoro**

### CODE REVIEW

File: mappings/net/minecraft/client/network/ClientPlayerEntity.mapping [hunk 1]
```diff
@@ -9,9 +9,18 @@ CLASS djf net/minecraft/client/network/ClientPlayerEntity
 	FIELD cd recipeBook Lcul;
 	FIELD ce tickables Ljava/util/List;
 	FIELD cf clientPermissionLevel I
+	FIELD cg lastX D
+	FIELD ch lastBaseY D
+	FIELD ci lastZ D
+	FIELD cj lastYaw F
+	FIELD ck lastPitch F
+	FIELD cl lastOnGround Z
+	FIELD cm lastIsHoldingSneakKey Z
```
Comment (Gegy) [inline mappings/net/minecraft/client/network/ClientPlayerEntity.mapping:18]: Could this be `lastSneaking`, to match `lastSprinting`, or is it treated differently?

File: mappings/net/minecraft/entity/LivingEntity.mapping [hunk 2]
```diff
@@ -76,28 +98,49 @@ CLASS aii net/minecraft/entity/LivingEntity
...
-	METHOD b getPotionEffect (Lahr;)Laht;
+	METHOD b applyArmorToDamage (Lahi;F)F
+		ARG 1 source
+		ARG 2 damage
+	METHOD b getStatusEffect (Lahr;)Laht;
 	METHOD b getEquippedStack (Laie;)Lbbq;
 	METHOD b getActiveEyeHeight (Lais;Laia;)F
 	METHOD b onEquipStack (Lbbq;)V
 		ARG 1 stack
+	METHOD b spawnEatingParticlesAndPlayEatingSoundOrPlayDrinkingSound (Lbbq;I)V
```
Comment (Gegy) [inline mappings/net/minecraft/entity/LivingEntity.mapping:109]: This is quite long.. 😄 
Could do something along the lines of `playConsumptionAnimation`

File: mappings/net/minecraft/entity/mob/IllagerEntity.mapping [hunk 3]
```diff
@@ -1,5 +1,6 @@
 CLASS ath net/minecraft/entity/mob/IllagerEntity
 	CLASS ath$a State
+	CLASS net/minecraft/entity/mob/IllagerEntity$b DoorInteractGoal
```
Comment (Gegy) [inline mappings/net/minecraft/entity/mob/IllagerEntity.mapping:3]: Knit bug

File: mappings/net/minecraft/entity/mob/SlimeEntity.mapping [hunk 4]
```diff
@@ -1,13 +1,37 @@
...
+		FIELD i targetYaw F
+		FIELD j ticksUntilJump I
 		FIELD k slime Lauk;
+		FIELD l jumpOften Z
+		METHOD a move (D)V
+			ARG 1 speed
+		METHOD a look (FZ)V
+			ARG 1 targetYaw
+			ARG 2 jumpOften
+	CLASS net/minecraft/entity/mob/SlimeEntity$a LookGoal
```
Comment (Gegy) [inline mappings/net/minecraft/entity/mob/SlimeEntity.mapping:12]: More Knit bugs here

File: mappings/net/minecraft/entity/mob/ZombieEntity.mapping [hunk 5]
```diff
@@ -1,5 +1,7 @@
 CLASS aus net/minecraft/entity/mob/ZombieEntity
 	CLASS aus$a DestroyEggGoal
+	CLASS net/minecraft/entity/mob/ZombieEntity$b ZombieEntityData
```
Comment (Gegy) [inline mappings/net/minecraft/entity/mob/ZombieEntity.mapping:3]: Here too

File: mappings/net/minecraft/entity/passive/DolphinEntity.mapping [hunk 6]
```diff
@@ -3,4 +3,6 @@ CLASS aqm net/minecraft/entity/passive/DolphinEntity
 		FIELD i dolphin Laqm;
 	CLASS aqm$c
 		FIELD a dolphinEntity Laqm;
+	CLASS net/minecraft/entity/passive/DolphinEntity$b
```
Comment (Gegy) [inline mappings/net/minecraft/entity/passive/DolphinEntity.mapping:6]: Also here

File: mappings/net/minecraft/entity/passive/LlamaEntity.mapping [hunk 7]
```diff
@@ -1,11 +1,20 @@
 CLASS arl net/minecraft/entity/passive/LlamaEntity
+	CLASS net/minecraft/entity/passive/LlamaEntity$b LlamaEntityData
```
Comment (Gegy) [inline mappings/net/minecraft/entity/passive/LlamaEntity.mapping:2]: Another Knit bug

File: mappings/net/minecraft/client/network/ClientPlayerEntity.mapping (hunk 1 above)
Comment (Runemoro) [reply mappings/net/minecraft/client/network/ClientPlayerEntity.mapping:18]: We already have lastSneaking, which is whether the player was sneaking last tick. This is whether the sneak key was down last tick (the player may be sneaking without holding sneak if it's in a <2 block tall space).

**Comment by modmuss50** [comment]
Merging due to update, open a new issue or PR with further comments.

**Pull request merged**